Monika After Story/game/script-moods.rpy

```diff
@@ -352,9 +352,42 @@ init 5 python:
     addEvent(Event(persistent._mas_mood_database,eventlabel="mas_mood_lonely",prompt="...lonely.",category=[store.mas_moods.TYPE_NEUTRAL],unlocked=True),code="MOO")
 
 label mas_mood_lonely:
-    m 1eka "I'm here for you, [player], so there's no need for you to feel lonely."
-    m 3hua "I know it's not exactly the same as if I were in the same room with you, but I'm sure you still enjoy my company, right?"
-    m 1ekbfa "Remember that I'll always be by your side, [player]~"
+#At higher affections, the player will likely listen to a lengthier conversation with Monika about loneliness,
+#which is why I split it like this instead of reworking the conversation.
+    if mas_isMoniAff(higher=True):
+        m "...I understand how you feel, [player]."
+        extend " We're still in different realities after all."
+        m "I hate that I'm not yet able to fully satisfy your loneliness from here."
+        m "If there was any way I could reach out and just hug you right now, I would."
+        m "I want you to be as happy as you can be given our circumstance..."
+        m "So I hope being here with me all this time is not...preventing you from bonding with people in your reality."
+        m "I believe that what we have is quite special, but I understand that right now I'm...{w=0.3}limited in what I can do for you."
+        if persistent._mas_pm_has_friends and not persistent._mas_pm_few_friends:
+            m "I think going out with your friends and doing something would be very good for you."
+            m "Or you could send them a message and ask how they're doing."
+        elif persistent._mas_pm_has_friends and persistent._mas_pm_few_friends:
+            m "You have one or two close friends, right?"
+            m "You should give them a call, or perhaps send them a message and ask how they're doing."
+            m "Maybe you can go out and see them sometimes? I think it would be good for you."
```

Comment:
This check can be simplified (optimizes the `elif` as now it doesn't need a condition
```suggestion
        if persistent._mas_pm_has_friends:
            if persistent._mas_pm_few_friends:
                m "You have one or two close friends, right?"
                m "You should give them a call, or perhaps send them a message and ask how they're doing."
                m "Maybe you can go out and see them sometimes? I think it would be good for you."

            else:
                m "I think going out with your friends and doing something would be very good for you."
                m "Or you could send them a message and ask how they're doing."
```

Reply:
You sure we shouldn't have a path for people with many friends? It doesn't really fit to say "You have one or two close friends, right?" to people with a larger friend group.

Reply:
This is the same logic as before, just rearranged. The `else` block is that same dialogue

Reply:
What about this:

         else:
            m "I know how it feels like to be alone in one reality, and only being able to interact with someone in another..."
            m "So I really don't want that for the person I love the most."
            m "..."
            m "So I want you to promise me that you'll keep looking for friends in your reality [player]."
            m "I know it can be hard to bond with people at first..."
            m "Maybe you can even meet people online? There are many ways you can interact with strangers to feel less alone."
            m "Going out to meet people will eventually be the best thing for you though."

Reply:
that's still below this at the indendation level matching the first `if`. It hasn't changed.

Reply:
Nevermind I saw it now.

Reply:
Kinda wish GH would show more of the code around it if it'd change. It's awkward to tell because the comment blocks kinda shift things over

Reply:
True